jipt/src/accent.ts

```diff
@@ -0,0 +1,44 @@
+import FrameListener from './frame-listener';
+import LiveNode from './mutation/live-node';
+import Mutation from './mutation/mutation';
+import State from './state';
+import Pin from './ui/pin';
+import UI from './ui/ui';
+
+export interface Config {
+  i: string;
+  h: string;
+}
```

Comment:
Is there a reason we shouldn’t use whole identifiers here?

```ts
projectID: string;
host: string
```

Reply:
Like Google analytics script, hotjar, mixpanel, etc we use the shortest variable name since the script is included in the user’s project.